shared/src/common/types/eam-group/eam-group-configurate-request-dto.ts

```diff
@@ -0,0 +1,6 @@
+type EAMGroupConfigurateRequestDto = {
+  name: string;
+  workers: string[];
```

Comment:
```suggestion
  workersIds: string[];
```
are these IDs right?